recipes/pyaerocom/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyaerocom" %}
+{% set version = "0.8.0rc1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8fcf2cf511cc11b3ff4bee81d742a0c8d67c8e5617590be844fbfbe769193a05
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyaeroeval=pyaerocom.web.cli.main_aerocom_evaluation:main
+    - pyaerotrends=pyaerocom.web.cli.main_trends_evaluation:main
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python >=3.7
+    - iris >=2.1.0
+    - xarray >=0.10.8
+    - cartopy >=0.16.0
+    - matplotlib >=3.0.1
+    - scipy >=1.1.0
+    - pandas >=0.23.0
+    - seaborn >=0.8.0
+    - netcdf4
+    - geonum
+    - cf_units
+    - numpy
+    - simplejson
+    - pytest
```

Comment:
I guess this isn't actually needed in `run`?
```suggestion
```
(if it is, it shouldn't be in the test requirements as it will be there regardless)

Reply:
True!